### ISSUE TITLE
The video!

### ISSUE THREAD

**Issue by logangbrown** [markhaslam/familysummit.org#9]
We fixed up the video, right? Where are we going to put that? Are we going to just link to YouTube for it? Maybe use the same account as the one that they will use for the calendars. I think I have the edited video somewhere if you don't have it.

**Comment by markhaslam** [comment]
Yeah I think you edited the video a bit to get rid of an old address or something. I don't think I have the video. If you can find it maybe you could upload it to the images folder on her? I think using YouTube sounds like a good idea as well as using the same account as the calendar. I guess if we didn't use YouTube we could just have the video hosted on the site... but there doesn't seem much reason for that that I can think of.

Also, looking at the handwritten homepage blueprint she gave, it looks like wants a box/button to click on to watch the video. If I remember right it looks like she wanted a link to it as well under the About Us down menu. If it helps, below is a screen snip from that page I am referring to. The whole page is in the PDF I scanned and emailed to you.

![image](https://cloud.githubusercontent.com/assets/6936232/9459611/d0221f5c-4aba-11e5-9623-187be49f4f78.png)
